CUBE_IDE/VGA/Core/Inc/API_functions.h

```diff
@@ -0,0 +1,23 @@
+/*******************************************************//**
+ * @file    API_functions.h
+ *
+ * @brief  	Header file for API functions
+ *
+ * @authors Naomi Born
+ * @date    17-05-2023
+ * @version 1.0
+*********************************************************/
+#ifndef INC_API_FUNCTIONS_H_
+#define INC_API_FUNCTIONS_H_
+
```

Comment:
no comments used? you did with the prototypes, but not the with the rest in the header file